session/handler.go

```diff
@@ -224,6 +237,200 @@ func (h *Handler) deleteIdentitySessions(w http.ResponseWriter, r *http.Request,
 	w.WriteHeader(http.StatusNoContent)
 }
 
+// swagger:parameters adminListIdentitySessions
+// nolint:deadcode,unused
+type adminListIdentitySessions struct {
+	// ActiveOnly is a boolean flag that filters out inactive sessions.
+	//
+	// default: false
+	// required: false
+	// in: query
+	ActiveOnly bool `json:"active_only"`
```

Comment:
What do you think about:

```suggestion
	Active bool `json:"active"`
```

and then setting it `true` or `false` will include active or not and if unset includes everything?

Reply:
ping @zepatrik

Reply:
Makes sense, fixed accordingly.